src/components/raycaster.js

```diff
@@ -60,25 +63,54 @@ module.exports.Component = registerComponent('raycaster', {
   },
 
   /**
+   * Update list of objects to test for intersection once child is loaded.
+   */
+  refreshOnceChildLoaded: function (evt) {
+    var self = this;
+    var el = evt.detail.el;
+    if (!el) { return; }
+    if (el.hasLoaded) { this.refreshObjects(); } else {
+      el.addEventListener('loaded', function nowRefresh (evt) {
+        el.removeEventListener('loaded', nowRefresh);
+        self.refreshObjects();
+      });
+    }
```

Comment:
looking at it a little more closely now, shouldn't this happen when children are attached somewhere in the scene?
`this.el.sceneEl.addEventListener('child-attached', this.refreshOnceChildLoaded);`